addons/trenches/CfgWorlds.hpp

```diff
@@ -0,0 +1,383 @@
+class CfgWorlds {
+    //Altis
+    class Altis {
+      surfaceTextureBasePath = "a3\map_data";
+      filePrefix = "co.paa";
+      camouflageObjects[] = {"A3\plants_f\Clutter\c_Thistle_Thorn_Brown.p3d"};
+    };
+
+    //Baranow
+    class Baranow {
+        surfaceTextureBasePath = "WW2\TerrainsIF_t\Worlds\IF_Surfaces_t\staszow";
```

Comment:
Are those vanilla config entries? If not, should be `GVAR`s.

Reply:
What would be the use of this?

Reply:
they're not. Ignore jonpas here.

Reply:
They're not. Ignore jonpas here.